src/util/update-dependency-version.util.ts

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from "fs/promises";
+
+import { formatCode } from "./format-code.util";
+
+export async function updateDependencyVersion(packageJsonPath: string, packageName: string, newVersion: string) {
+    const fileContent = (await readFile(packageJsonPath)).toString();
+    const packageJson = JSON.parse(fileContent);
```

Comment:
Should we check if the file exists? Otherwise this might fail if there's no Admin

Reply:
Good point, done in 00bfbf2eea89b2def42ceacb0f892e9de0f60243